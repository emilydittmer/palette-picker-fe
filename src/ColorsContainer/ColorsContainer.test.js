import React from "react";
import {
  ColorsContainer,
  mapStateToProps,
  mapDispatchToProps
} from "./ColorsContainer";
import { 
  addPalette, 
  savePalette, 
  getPalettes 
} from "../Actions";
import { shallow } from 'enzyme'

describe("ColorsContainer", () => {
  let wrapper, props;
  beforeEach(() => {
    props = {
      palettes: [],
      projects:  [{id: 1, title: 'Example 1'}, {id: 2, title: 'Example 2'}],
      currentPalette: []
    };
    wrapper = shallow(<ColorsContainer {...props} />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
