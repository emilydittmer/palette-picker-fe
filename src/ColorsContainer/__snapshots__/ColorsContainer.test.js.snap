// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorsContainer should match snapshot 1`] = `ShallowWrapper {}`;

exports[`ColorsContainer should set state for the current project 1`] = `ShallowWrapper {}`;

exports[`ColorsContainer should set state for the error message 1`] = `ShallowWrapper {}`;

exports[`ColorsContainer should set state for the locked state 1`] = `ShallowWrapper {}`;
